src/components/container/SurveyAnswerChart/SurveyAnswerChart.tsx

```diff
@@ -183,5 +193,41 @@ export default function SurveyAnswerChart(props:SurveyAnswerChartProps) {
         tooltip={GraphToolTip}
         chartType={props.chartType}
         options={props.options}
+        innerRef={props.innerRef}
     />
+}
+
+export async function generateSurveyResponse(date: Date, resultIdentifier: string, surveyName: string, rangeStart: number, rangeEnd: number): Promise<SurveyAnswer> {
```

Comment:
Is there a reason not to put the preview data generation into a .previewdata.tsx file?

Reply:
I missed that pattern in our repo!  Appreciate the heads up, I've moved the functions that generate this.